docs/3_finetuner_in_action/3_1_text_to_text.md

```diff
@@ -1,3 +1,118 @@
 # Text to text search using Bert
 
-Coming soon!
\ No newline at end of file
+Searching large amounts of text documents with text queries is a very popular use-case, so of course Finetuner enables you to accomplish this easily.
+
+This guide will lead you through an example use-case to show you how Finetuner can be used for text-to-text retrieval.
+
+In Finetuner, two models are supported as backbones, namely `bert-base-cased` and `sentence-transformers/msmarco-distilbert-base-v3`, both of which are models hosted on Hugging Face.
+
+In this example, we will fine-tune `sentence-transformers/msmarco-distilbert-base-v3` on the [Quora Question Pairs dataset](https://www.kaggle.com/competitions/quora-question-pairs), where the search task involves finding duplicate questions in the dataset. An example query for this search task might look as follows:
```

Comment:
can we fine-tune bert rather than sentence transformers? I think they already have [sentence-transformers fine-tuned on quoraqa](https://huggingface.co/sentence-transformers/quora-distilbert-multilingual), i don't know

Reply:
sure thing